Features and fixes

projected1 opened the pull request:
Added closeOnBlur option so users can choose not to close the overlay when the overlay itself is clicked. The implementation is smart enough to discard this flag in case a close button in not provided, so that one won't get stuck with the overlay open.
Fixed the code, so that external CSS is not required anymore.

projected1 committed:
Minify the JavaScript

projected1 committed:
Add closeOnBlur option and fix overlay so no external CSS is required

projected1 committed:
Change dialog position from "fixed" to "absolute"

If the dialog is larger than the browser window, it's impossible
to scroll to the bottom of it if its positioning is "fixed".

projected1 committed:
Minify

cbeams commented:
When using the minified version of the JS here, I get the following error:

```
Uncaught SyntaxError: Unexpected token var jquery.leanModal.min.js:1
```

There is no such problem when using the non-minified version.

projected1 committed:
Minify (fixed)

projected1 commented:
Fixed the minified script.

samael83 committed:
change top offset unit type from 'PX' to user define.

projected1 committed:
Merge pull request #1 from Samael803/master

change top offset unit type from 'PX' to user define.

projected1 committed:
Minify

projected1 committed:
Add overlay color property

projected1 committed:
Use jQuery show/hide instead of manually overriding CSS display

projected1 committed:
Change positioning from 'absolute' to 'fixed'

We want the modal element to be immediately visible when shown so
the user does not need to scroll and look for it.
